mesa/mesaPROTON_OC.py

```diff
@@ -1166,6 +1176,136 @@ def let_migrants_in(self):
                 new_agent.wealth_level = job.job_level
                 new_agent.migrant = True
 
+    def commit_crimes(self):
+        """
+        This procedure is central in the model, allowing agents to find accomplices and commit crimes.
+        Based on the table self.c_range_by_age_and_sex , the number of crimes and the subset of the agents who commit them
+        is selected. For each crime a single agent is selected and if necessary activates the procedure that allows
+        the agent to find accomplices. Criminal groups are append within the list self.co_offender_groups
```

Comment:
"are append within" makes no sense

Reply:
All docstrings have been refined in a future PR.